bundle/jsky.app.ot/src/main/java/jsky/app/ot/editor/seq/DynamicSequenceTableModel.java

```diff
@@ -40,22 +36,22 @@
 
     private SPNodeKey _nodeKey;
 
-    public void setSequence(ConfigSequence sequence, SPNodeKey key, List<ItemKey> alwaysShow) {
+    public void setSequence(final ConfigSequence sequence, final SPNodeKey key, final List<ItemKey> alwaysShow) {
         _sequence     = sequence;
         _nodeKey      = key;
 
         final ItemKey[] newIteratedKeys = sort(getIteratedKeys(sequence, alwaysShow));
-        if (newIteratedKeys.equals(_iteratedKeys)) {
+        if (Arrays.equals(newIteratedKeys, _iteratedKeys)) {
             fireTableDataChanged();
         } else {
-            _iteratedKeys = sort(getIteratedKeys(sequence, alwaysShow));
+            _iteratedKeys = newIteratedKeys;
```

Comment:
`sort(getIteratedKeys(sequence, alwaysShow))` seems to be side-effect free, so I think we don't need to evaluate this twice.